docs/roadmap/binary-search/binary-search-problems.md

```diff
@@ -25,8 +25,10 @@ description: Binary Search practice problems by cyberlabs
 16. [448/D - Multiplication Table](https://codeforces.com/problemset/problem/448/D)
 17. [779/D - String Game](https://codeforces.com/contest/779/problem/D)
 18. [1029/F - Multicolored Markers](https://codeforces.com/contest/1029/problem/F)
+19. [1409/E  - Two Platforms](https://codeforces.com/problemset/problem/1409/E)
```

Comment:
Thanks for the contributions!
It would be better if we had consistent spacing conventions. Could you please update this line to have only one space between `1409/E` and `- Two Platforms` instead of two?